### PR TITLE
Complete website

### PR DESCRIPTION
**Pull request by zied2112**
In this milestone we did the following:

-Add Navbar and footer
-Separate three content section called by the navbar
-Add Synchronous date and time

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -75,3 +111,24 @@ input {
...
+.cont-info {
+  text-align: center;
+}
+
+.cont-list {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
```
Comment (binhussen) [inline css/style.css]: - Great job for implementing the navigation :+1: however you missed the wireframe at some point, and your submission form and the table list aren't followed the wireframe, please update to have a better design and fulfill the [requirements](https://github.com/microverseinc/curriculum-javascript/blob/main/books/m4_full_website_v1_1.md#project-requirements) :+1: 

#### given Wireframe
Screenshot from 2023-02-15 23-47-18
#### your design
Screenshot from 2023-02-15 23-46-33
Screenshot from 2023-02-15 23-46-20

**Review by binhussen — CHANGES_REQUESTED** [review]
Hi @zied2112 and @Kevin-Mena , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights

- All linter checks passed ✔️
- Descriptive pull request ✔️
- Used GitHub flow ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: css/style.css (hunk 1 above)
Comment (zied2112) [reply css/style.css]: Hello @binhussen, After fixing that we had some problems with the linters, so if you can help us we will be so grateful

**Comment by zied2112** [comment]
image

**Comment by zied2112** [comment]
image

**Comment by prolajumokeoni** [comment]
Hi @zied2112 and @Kevin-Mena, could you try deleting node modules locally and try running the stylelint linter installation locally again

**Comment by zied2112** [comment]
I did it, nothing changes

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Review by EricMbouwe — APPROVED** [review]
Hi @zied2112, 

Your project is complete! ⚡🔥  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 🎉🔥
- Requested changes ✔
- No linter errors ✔

## Optional suggestions ☂

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**